scripts/create-user.sh

```diff
@@ -18,7 +18,7 @@ MESSAGE='Hello,
 Your account for GOV.UK PaaS is ready:
 
  - username: ${EMAIL}
- - organisation: ${ORG}
+${ORG:+ - organisation: ${ORG}}
```

Comment:
It's possible. I'm not sure about the non-terrible part. Some examples:

```sh
 - username: ${EMAIL}${ORG:+\n - organisation: ${ORG}}
```

```sh
 - username: ${EMAIL}${ORG:+
 - organisation: ${ORG}}
```

What do you think?